CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Javis.jl - Changelog
 
+## Unreleased v0.2
```

Comment:
I think you created the branch from the wrong base branch 😉

Reply:
Yea... Just realized that I did that... Fixed now however!